pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the LightningTensor class that inherits from the new device interface.
+It is a device to perform tensor network simulation of a quantum circuit. 
+"""
+from dataclasses import replace
+from numbers import Number
+from typing import Callable, Optional, Sequence, Tuple, Union
+
+import numpy as np
+import pennylane as qml
+from pennylane.devices import DefaultExecutionConfig, Device, ExecutionConfig
+from pennylane.devices.modifiers import simulator_tracking, single_tape_support
+from pennylane.tape import QuantumTape
+from pennylane.transforms.core import TransformProgram
+from pennylane.typing import Result, ResultBatch
+
+from .quimb._mps import QuimbMPS
```

Comment:
Can this directory be made a module and we can import the `QuimbMPS` class from there? If making `quimb` a module is problematic due to the already existing `quimb` package name, then maybe we can modify this directory name to something like `backends`, and have the class live in there,

Reply:
Ok, I'll create a `backends` folder and move the `quimb` folder inside it